src/components/grid-view/__tests__/__snapshots__/GridView.test.js.snap

```diff
@@ -37,6 +37,7 @@ exports[`components/grid-view/GridView should render() 1`] = `
     flexGrow={1}
     flexShrink={1}
     headerRenderer={[Function]}
+    headerRole="columnheader"
```

Comment:
previously it was hard coded into react-virtualized/table, and now we are adding the hard-coded value as a default prop that can be overridden